### ISSUE TITLE
InfraCost `--output` Produces Empty HTML Report

### ISSUE THREAD

**Issue by AErmie** [infracost/infracost#1012]
Though using the example from the [Generate reports](https://www.infracost.io/docs/multi_project/report) documentation, my resulting HTML file is blank, even though the JSON file has populated values.

Am I missing some property, or using the wrong commands for the report generation? 

![InfraCost-BlankHTML](https://user-images.githubusercontent.com/10376796/135733634-710c247d-27a7-4eed-ac5d-cc1578748888.png)

**Example code:**

```yaml
mkdir InfraCostReport

docker run --rm \
-e INFRACOST_API_KEY=$(InfraCost_API_KEY) \
-e AWS_ACCESS_KEY_ID=$(AccessKeyID_DEV) \
-e AWS_SECRET_ACCESS_KEY=$(SecretAccessKey_DEV) \
--volume $(System.DefaultWorkingDirectory)/InfraCostReport:/InfraCostReport/ \
--volume $(System.DefaultWorkingDirectory):/code/ \
--workdir /code/terraform \
infracost/infracost breakdown --path /code/terraform/TerraformPlan.json --format json > InfraCostReport/Transcoding-Breakdown.json

docker run --rm \
-e INFRACOST_API_KEY=$(InfraCost_API_KEY) \
-e AWS_ACCESS_KEY_ID=$(AccessKeyID_DEV) \
-e AWS_SECRET_ACCESS_KEY=$(SecretAccessKey_DEV) \
--volume $(System.DefaultWorkingDirectory)/InfraCostReport:/InfraCostReport/ \
--volume $(System.DefaultWorkingDirectory):/code/ \
--workdir /code/terraform \
infracost/infracost output --path InfraCostReport/Transcoding-Breakdown.json --format html > InfraCostReport/Transcoding-Breakdown.html
```

**Comment by aliscott** [comment]
@AErmie your commands look correct. Are you seeing any output when running with the table output format? e.g.

```
docker run --rm \
-e INFRACOST_API_KEY=$(InfraCost_API_KEY) \
-e AWS_ACCESS_KEY_ID=$(AccessKeyID_DEV) \
-e AWS_SECRET_ACCESS_KEY=$(SecretAccessKey_DEV) \
--volume $(System.DefaultWorkingDirectory)/InfraCostReport:/InfraCostReport/ \
--volume $(System.DefaultWorkingDirectory):/code/ \
--workdir /code/terraform \
infracost/infracost output --path InfraCostReport/Transcoding-Breakdown.json --format table
```

If not, are you able to provide a sample of the JSON file?

**Comment by aliscott** [comment]
Thanks @AErmie, you can email it to me at alistair@infracost.io.

**Comment by aliscott** [comment]
Thanks @AErmie I wonder if this is the same issue with the relative path not existing. Does it work if you run `infracost/infracost output --path /InfraCostReport/Transcoding-Breakdown.json`?

Side note: thanks to you sending this file I found another bug here when passing in `~` in the path. I've pushed this in https://github.com/infracost/infracost/pull/1017 which will also now display an error if the a`--path` cannot be found instead of just the empty report you are seeing.
